snapcraft.yaml

```diff
@@ -0,0 +1,49 @@
+name: wksctl
+version: git
```

Comment:
https://github.com/fluxcd/flux/blame/master/snap/snapcraft.yaml#L23-L37 might be helpful for automatically getting you the version.